timesketch/lib/analyzers/phishy_domains.py

```diff
@@ -244,9 +253,9 @@ def run(self):
                 text = 'Domain {0:s} is similar to {1:s}'.format(
                     domain, ', '.join(similar_text_list))
                 if any(domain.endswith(
-                        x) for x in self.domain_scoring_whitelist):
-                    tags_to_add.append('whitelisted-domain')
-                    whitelist_encountered = True
+                        x) for x in self.domain_scoring_exclude_domains):
+                    tags_to_add.append('allowlisted-domain')
```

Comment:
s/allowlisted-domain/known-domain

Reply:
LGTM, done in https://github.com/google/timesketch/pull/1245/commits/1989e31b5ea2efadfe816fa0ed739270c1d07483